Battery/AccelerometerSDK.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Accelerometer } from 'expo-sensors';
+import BatteryIcon from './BatteryIcon';
+
+export function AccelerometerSDK() {
+  const [batteryLevel, setBatteryLevel] = useState(0);
+  const [subscription, setSubscription] = useState<Accelerometer.Subscription | null>(null);
+
+  const _subscribe = () => {
+    setSubscription(
+      Accelerometer.addListener(({ x }) => {
+        if (Math.abs(x) > 3) {
```

Comment:
3 is really high. I thought the app was broken at first. You have to shake it as hard as possible to register 3 g's. Someone will have a stroke trying to get this to work 😆